### PR TITLE
Add inline cdr doc

### PR DESCRIPTION
**Pull request by gsanchietti**
Add inline doc for CDR.
After the merge, the package should be ready for official release.

### CODE REVIEW

**Comment by gsanchietti** [comment]
Replaces #119

File: ui/src/doc-inline/it/cdr/dashboard_default_9_pie_inboundAvgDurationByDid.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+I DID (Direct Inward Dial) sono le numerazioni in entrata configurate e gestite
+dal centralino telefonico.
+
+Per durata si intende il periodo di tempo, visualizzato in secondi/minuti/ore, di
+svolgimento della telefonata dal suo arrivo alla sua chiusura.
+
+Se nessun DID è stato selezionato, il grafico mostrerà tutti i DID disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I DID eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_9_pie_inboundAvgDurationByDid.md:17]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_8_pie_inboundCallsByDid.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Il grafico a torta mostra il totale delle chiamate entrate nei DID selezionati
+e le mette a confronto in modo proporzionale.
+
+I DID (Direct Inward Dial) sono le numerazioni in entrata configurate e gestite 
+dal centralino telefonico.
+
+Se nessun DID è stato selezionato, il grafico mostrerà tutti i DID disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I DID eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_8_pie_inboundCallsByDid.md:14]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_7_pie_inboundProvinces.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Chiamate in Entrata per Provincia
+
+Numero di chiamate *in Entrata* per i Fasci suddiviso per provenienza
+su base provinciale.
+
+Il grafico a torta mostra il numero delle chiamate entrate dai fasci valutando 
+la provenienza su base provinciale e le mette a confronto in modo proporzionale.
+
+Il grafico può contenere al massimo 8 sezioni. Le province eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_7_pie_inboundProvinces.md:10]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_6_pie_inboundRegions.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+Numero di chiamate *in Entrata* per i Fasci suddiviso per provenienza su base 
+regionale.
+
+Il grafico a torta mostra il numero delle chiamate entrate dai fasci valutando 
+la provenienza su base regionale e le mette a confronto 
+in modo proporzionale.
+
+Il grafico può contenere al massimo 8 sezioni. Le regioni eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_6_pie_inboundRegions.md:11]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_5_bar_inboundMonthlyDistribution.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+Distribuzione mensile delle chiamate *in Entrata* per i Fasci selezionati.
+
+Il grafico mostra per i mesi dell'intervallo impostato il totale delle chiamate in
+entrata dai fasci selezionati.
+
+Se nessun fascio è stato selezionato, il grafico mostrerà tutti i fasci disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I fasci eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_5_bar_inboundMonthlyDistribution.md:11]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_7_pie_inboundProvinces.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+Se presente, cliccare il bottone `Visualizza dettagli` per visualizzare tutti i dati
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_7_pie_inboundProvinces.md]:
```suggestion
Se presente, cliccare il bottone `Mostra dettagli` per visualizzare tutti i dati
```

File: ui/src/doc-inline/it/cdr/dashboard_default_6_pie_inboundRegions.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+Se presente, cliccare il bottone `Visualizza dettagli` per visualizzare tutti i dati
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_6_pie_inboundRegions.md]:
```suggestion
Se presente, cliccare il bottone `Mostra dettagli` per visualizzare tutti i dati
```

File: ui/src/doc-inline/it/cdr/dashboard_default_5_bar_inboundMonthlyDistribution.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+Se presente, cliccare il bottone `Visualizza dettagli` per visualizzare tutti i dati
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_5_bar_inboundMonthlyDistribution.md]:
```suggestion
Se presente, cliccare il bottone `Mostra dettagli` per visualizzare tutti i dati
```

File: ui/src/doc-inline/it/cdr/dashboard_default_4_bar_inboundHourlyDistribution.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+Distribuzione oraria delle chiamate *in Entrata* per i Fasci selezionati.
+
+Il grafico mostra nell'orario lavorativo impostato il totale delle chiamate in 
+entrata dai fasci selezionati.
+
+Se nessun fascio è stato selezionato, il grafico mostrerà tutti i fasci disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I fasci eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_4_bar_inboundHourlyDistribution.md:11]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_4_bar_inboundHourlyDistribution.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+Se presente, cliccare il bottone `Visualizza dettagli` per visualizzare tutti i dati
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_4_bar_inboundHourlyDistribution.md]:
```suggestion
Se presente, cliccare il bottone `Mostra dettagli` per visualizzare tutti i dati
```

File: ui/src/doc-inline/it/cdr/dashboard_default_3_pie_inboundAvgWait.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+Il grafico a torta mostra l'attesa media delle chiamate entrate dai fasci
+selezionati e le mette a confronto in modo proporzionale.
+
+Per *attesa* si intende il tempo trascorso dall'arrivo della chiamata alla sua
+risposta.
+
+Se nessun fascio è stato selezionato, il grafico mostrerà tutti i fasci disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I fasci eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_3_pie_inboundAvgWait.md:14]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_2_pie_inboundAvgDuration.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+Il grafico a torta mostra la durata media delle chiamate entrate dai fasci selezionati
+e le mette a confronto in modo proporzionale.
+
+Per durata si intende il periodo di tempo, visualizzato in secondi/minuti/ore, di 
+svolgimento della telefonata dal suo arrivo alla sua chiusura.
+
+Se nessun fascio è stato selezionato, il grafico mostrerà tutti i fasci disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I fasci eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_2_pie_inboundAvgDuration.md:14]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_1_pie_inboundCalls.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+# Chiamate in Entrata per Fascio
+
+Numero di chiamate totali *in Entrata* per Fasci selezionati.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_1_pie_inboundCalls.md:3]:
```suggestion
Numero di chiamate totali *in Entrata* per i Fasci selezionati.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_1_pie_inboundCalls.md [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+
+Il grafico a torta mostra il totale delle chiamate entrate dai fasci selezionati
+e le mette a confronto in modo proporzionale.
+
+Se nessun fascio è stato selezionato, il grafico mostrerà tutti i fasci disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I fasci eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_1_pie_inboundCalls.md:11]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_18_pie_outboundProvinces.md [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+
+Numero di chiamate *in Uscita* per i Fasci suddiviso per provenienza
+su base provinciale.
+
+Il grafico a torta mostra il numero delle chiamate entrate dai fasci
+ valutando la provenienza su base provinciale e le mette a confronto
+in modo proporzionale.
+
+Il grafico può contenere al massimo 8 sezioni. Le province eccedenti 
+questo numero verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_18_pie_outboundProvinces.md:11]:
```suggestion
questo numero verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_18_pie_outboundProvinces.md [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+
+Se presente, cliccare il bottone `Visualizza dettagli` per
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_18_pie_outboundProvinces.md]:
```suggestion
Se presente, cliccare il bottone `Mostra dettagli` per 
```

File: ui/src/doc-inline/it/cdr/dashboard_default_17_pie_outboundRegions.md [hunk 17]
```diff
@@ -0,0 +1,14 @@
...
+
+Numero di chiamate *in Uscita* per i Fasci suddiviso per provenienza
+su base regionale.
+
+Il grafico a torta mostra il numero delle chiamate entrate dai fasci
+ valutando la provenienza su base regionale e le mette a confronto
+in modo proporzionale.
+
+Il grafico può contenere al massimo 8 sezioni. Le regioni eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_17_pie_outboundRegions.md:11]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_17_pie_outboundRegions.md [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+
+Se presente, cliccare il bottone `Visualizza dettagli` per visualizzare tutti i dati
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_17_pie_outboundRegions.md]:
```suggestion
Se presente, cliccare il bottone `Mostra dettagli` per visualizzare tutti i dati
```

File: ui/src/doc-inline/it/cdr/dashboard_default_16_pie_outboundCostByGroup.md [hunk 19]
```diff
@@ -0,0 +1,13 @@
+# Costo Chiamate in Uscita per Gruppo
+
+Costo delle chiamate *in Uscita* per i Gruppi di Utenti.
+
+Il grafico a torta mostra il costo delle chiamate effettuate dagli utenti dei
+gruppi configurati sul centralino telefonico e le mette a confronto in modo
+proporzionale.
+
+Il grafico può contenere al massimo 8 sezioni. I gruppi eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_16_pie_outboundCostByGroup.md:10]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_15_pie_outboundCallsByGroup.md [hunk 20]
```diff
@@ -0,0 +1,14 @@
+# Chiamate in Uscita per Gruppo di Utenti
+
+Numero di chiamate totali *in Uscita* per i Gruppi di Utenti.
+
+Il grafico a torta mostra il totale delle chiamate effettuate dagli utenti dei 
+gruppi configurati sul centralino telefonico e le mette a confronto in modo 
+proporzionale.
+
+Il grafico può contenere al massimo 8 sezioni. I gruppi eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_15_pie_outboundCallsByGroup.md:10]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_14_pie_outboundCallsByRoute.md [hunk 21]
```diff
@@ -0,0 +1,15 @@
+# Chiamate in Uscita per Fascio
+
+Numero di chiamate totali *in Uscita* per Fasci configurati.
+
+Il grafico a torta mostra il totale delle chiamate uscite dai fasci e le mette 
+a confronto in modo proporzionale.
+
+Il grafico mostrerà tutti i fasci disponibili.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_14_pie_outboundCallsByRoute.md:8]:
```suggestion
Il grafico mostra tutti i fasci disponibili.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_14_pie_outboundCallsByRoute.md [hunk 22]
```diff
@@ -0,0 +1,15 @@
...
+
+Il grafico può contenere al massimo 8 sezioni. I fasci eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_14_pie_outboundCallsByRoute.md:11]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_12_bar_inboundMonthlyDistributionByDid.md [hunk 23]
```diff
@@ -0,0 +1,18 @@
...
+Il grafico mostra per i mesi dell'intervallo impostato il totale delle chiamate in
+entrata dai DID selezionati.
+
+I DID (Direct Inward Dial) sono le numerazioni in entrata configurate e gestite
+dal centralino telefonico.
+
+Se nessun DID è stato selezionato, il grafico mostrerà tutti i DID disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I DID eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_12_bar_inboundMonthlyDistributionByDid.md:14]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_11_bar_inboundHourlyDistributionByDid.md [hunk 24]
```diff
@@ -0,0 +1,18 @@
...
+Il grafico mostra nell'orario lavorativo impostato il totale delle chiamate in
+entrata dai DID selezionati.
+
+I DID (Direct Inward Dial) sono le numerazioni in entrata configurate e gestite
+dal centralino telefonico.
+
+Se nessun DID è stato selezionato, il grafico mostrerà tutti i DID disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I DID eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_11_bar_inboundHourlyDistributionByDid.md:14]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

File: ui/src/doc-inline/it/cdr/dashboard_default_12_bar_inboundMonthlyDistributionByDid.md [hunk 25]
```diff
@@ -0,0 +1,18 @@
...
+
+Se presente, cliccare il bottone `Visualizza dettagli` per visualizzare tutti i dati
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_12_bar_inboundMonthlyDistributionByDid.md]:
```suggestion
Se presente, cliccare il bottone `Mostra dettagli` per visualizzare tutti i dati
```

File: ui/src/doc-inline/it/cdr/dashboard_default_11_bar_inboundHourlyDistributionByDid.md [hunk 26]
```diff
@@ -0,0 +1,18 @@
...
+
+Se presente, cliccare il bottone `Visualizza dettagli` per visualizzare tutti i dati
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_11_bar_inboundHourlyDistributionByDid.md]:
```suggestion
Se presente, cliccare il bottone `Mostra dettagli` per visualizzare tutti i dati
```

File: ui/src/doc-inline/it/cdr/dashboard_default_10_pie_inboundAvgWaitByDid.md [hunk 27]
```diff
@@ -0,0 +1,20 @@
...
+I DID (Direct Inward Dial) sono le numerazioni in entrata configurate e gestite
+dal centralino telefonico.
+
+Per *attesa* si intende il tempo trascorso dall'arrivo della chiamata alla sua
+risposta.
+
+Se nessun DID è stato selezionato, il grafico mostrerà tutti i DID disponibili.
+
+Il grafico può contenere al massimo 8 sezioni. I DID eccedenti questo numero
+verranno visualizzate in una sezione `Altri`.
```
Comment (andre8244) [inline ui/src/doc-inline/it/cdr/dashboard_default_10_pie_inboundAvgWaitByDid.md:17]:
```suggestion
verranno visualizzati in una sezione `Altri`.
```

**Review by andre8244 — CHANGES_REQUESTED** [review]

File: ui/src/doc-inline/en/cdr/dashboard_default.md [hunk 28]
```diff
@@ -1 +1,3 @@
-This view provides a brief overview on **CDR & cost report**
+This view offers an overview of the **Calls and Costs Report**. The data from the current day is 
+always excluded.
+This view displays information on the switchboard Inbound calls.
```
Comment (andre8244) [inline ui/src/doc-inline/en/cdr/dashboard_default.md:3]: This line refers to "Inbound calls" and is not present in the italian version. Should we remove it?

**Review by andre8244 — APPROVED** [review]
LGTM

**Pull request merged**